control-plane/subcommand/gateway-resources/command.go

```diff
@@ -159,8 +174,14 @@ func (c *Command) Run(args []string) int {
 		return 1
 	}
 
-	// Load config from the configmap.
-	if err := c.loadConfig(); err != nil {
+	// Load apigw resource config from the configmap.
+	if err := c.loadAPIGWResourceConfig(); err != nil {
+		c.UI.Error(fmt.Sprintf("Error loading config: %s", err))
+		return 1
+	}
+
+	// Load mesh gateway config from the configmap.
+	if err := c.loadMeshGatewayConfigs(); err != nil {
 		c.UI.Error(fmt.Sprintf("Error loading config: %s", err))
```

Comment:
~ update error message `c.UI.Error(fmt.Sprintf("Error loading mesh gw config: %s", err))`

Reply:
Similarly here, the config file we're working with will contain/configure other gateway types in the future